napari/layers/points/_points_mouse_bindings.py

```diff
@@ -111,13 +111,19 @@ def select(layer, event):
 
 def add(layer, event):
     """Add a new point at the clicked position."""
+    start_pos = event.pos
+    dist = 0
+    yield
 
-    if event.type == 'mouse_press':
-        start_pos = event.pos
-
-    while event.type != 'mouse_release':
+    while event.type == 'mouse_move':
+        dist = np.linalg.norm(start_pos - event.pos)
```

Comment:
Like print this `dist` and test how it behaves when you perform long and short moves.